.github/workflows/build.yml

```diff
@@ -0,0 +1,72 @@
+name: pytorch-widedeep
+
+on:
+  push:
+#    branches: [ $default-branch ]
+  pull_request:
+#    branches: [ $default-branch ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: [3.7.9, 3.8, 3.9]
```

Comment:
I specified `3.7.9` because Travis was driving me crazy.  I think we can just leave it as `3.7`. Also, I always test 3.6 locally simply to save credits in Travis. Since Github Actions is free we should do: 

```yaml
python-version: [3.6, 3.7, 3.8, 3.9]
```

In fact, I intend to drop support for 3.6. sometime soon

Reply:
fixed in new commit